server/runtime/src/main/java/org/infinispan/server/Bootstrap.java

```diff
@@ -107,6 +107,12 @@ protected void handleArgumentCommand(String command, String parameter, Iterator<
             if (!properties.containsKey(Server.JGROUPS_BIND_PORT)) {
                properties.setProperty(Server.JGROUPS_BIND_PORT, Integer.toString(Server.DEFAULT_JGROUPS_BIND_PORT + offset));
             }
+            if (!properties.containsKey(Server.JGROUPS_FD_TCP_BIND_PORT)) {
```

Comment:
(I need to test this)
if I understand correctly, if the user sets `-Djgroups.fd.port=xxxx`  & `--offset=xxxx`, the `jgroups.fd.port` is ignored. is that the intention?

Reply:
No, if the user sets `-Djgroups.fd.port=xxxx` then it won't apply the offset

Reply:
no, the offset only checks `jgroups.fd.tcp.port` and `jgroups.fd.udp.port`, and It never checks `jgroups.fd.port`. And the configuration (`<FD_SOCK start_port="${jgroups.fd.tcp.port,jgroups.fd.port:57600}"/>`) first checks `jgroups.fd.tcp.port` and then `jgroups.fd.port`. One more reason I don't like to have multiple properties for the same xml attribute :(

I did the following test:
```
$ ./bin/server.sh -Djgroups.fd.port=9999
$ sudo ss -tlpn | grep pid=940614
LISTEN 0      4096            127.0.0.1:11222      0.0.0.0:*    users:(("java",pid=940614,fd=141))
LISTEN 0      50          192.168.2.100:7800       0.0.0.0:*    users:(("java",pid=940614,fd=106))
LISTEN 0      50          192.168.2.100:9999       0.0.0.0:*    users:(("java",pid=940614,fd=109))

$ ./bin/server.sh -Djgroups.fd.port=9999 -o 100
$ sudo ss -tlpn | grep pid=941806
LISTEN 0      4096            127.0.0.1:11322      0.0.0.0:*    users:(("java",pid=941806,fd=114))
LISTEN 0      50          192.168.2.100:7900       0.0.0.0:*    users:(("java",pid=941806,fd=106))
LISTEN 0      50          192.168.2.100:57700      0.0.0.0:*    users:(("java",pid=941806,fd=109))
```

Reply:
and if you use `-Djgroups.fd.tcp.port=9999`, offset is ignored:

```
$ ./bin/server.sh -Djgroups.fd.tcp.port=9999 -o 100
$ sudo ss -tlpn | grep pid=942877
[sudo] password for pedro: 
LISTEN 0      4096            127.0.0.1:11322      0.0.0.0:*    users:(("java",pid=942877,fd=141))
LISTEN 0      50          192.168.2.100:7900       0.0.0.0:*    users:(("java",pid=942877,fd=106))
LISTEN 0      50          192.168.2.100:9999       0.0.0.0:*    users:(("java",pid=942877,fd=109))
```